dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -425,6 +442,92 @@ enum NetworkCommands {
     ListVnics,
 }
 
+#[derive(Debug, Args)]
+struct RegionArgs {
+    #[command(subcommand)]
+    command: RegionCommands,
+}
+
+#[derive(Debug, Subcommand)]
+enum RegionCommands {
+    /// List all regions
+    List(RegionListArgs),
+
+    /// Find what is using a region
+    UsedBy(RegionUsedByArgs),
+
+    /// Find deleted volume regions
+    FindDeletedVolumeRegions,
+}
+
+#[derive(Debug, Args)]
+struct RegionListArgs {
+    /// Print region IDs only
+    #[arg(short)]
+    id_only: bool,
+}
+
+#[derive(Debug, Args)]
+struct RegionUsedByArgs {
+    region_id: Vec<Uuid>,
+}
+
+#[derive(Debug, Args)]
+struct SagaArgs {
+    #[command(subcommand)]
+    command: SagaCommands,
+}
+
+#[derive(Debug, Subcommand)]
```

Comment:
Just a suggestion: this file is getting pretty huge and I think we should start splitting it up in modules in the same way that the top-level omdb commands are split into modules.  I did that in https://github.com/oxidecomputer/omicron/pull/7695.  Basically, I'd put your new subcommands into `db/saga.rs` and `db/region.rs` and only expose the subcommand enums from them.